cmd/celestia/bridge.go

```diff
@@ -35,6 +35,15 @@ func init() {
 			gateway.Flags(),
 			state.Flags(),
 		),
+		authCmd(
+			cmdnode.NodeFlags(),
+			p2p.Flags(),
+			core.Flags(),
+			cmdnode.MiscFlags(),
+			rpc.Flags(),
+			gateway.Flags(),
+			state.Flags(),
+		),
```

Comment:
* Why authCmd needs all these flags? It seems like it doesn't
* AuthCmd should be located in `cmd` not `cmd/celestia`

Reply:
It needs to be able to parse node store flags, p2p flags, etc. Also persistent pre run on bridge, light and full cmds.